quote arguments to avoid problems with spaces in directory

here is what happens when running from folder with a space:

```
jbang /Users/max/Dropbox/My Mac (Maxs-MBP-2.localdomain)/Downloads/demo/gg/fetch.java
❯ jbang /Users/max/Dropbox/My Mac (Maxs-MBP-2.localdomain)/Downloads/demo/gg/fetch.java
[jbang] [ERROR] Script or alias could not be found or read: '/Users/max/Dropbox/My'
[jbang] Run with --verbose for more details
```

should have been ` jbang "/Users/max/Dropbox/My Mac (Maxs-MBP-2.localdomain)/Downloads/demo/gg/fetch.java"`

--- a/src/JBangRunner.ts
+++ b/src/JBangRunner.ts
@@ -17,7 +17,7 @@ export class JBangRunner {
 
     public async runJBang(uri: Uri): Promise<Terminal> {
         const exec = jbang();
-        const command = `${exec} ${uri.fsPath}`;
+        const command = `${exec} "${uri.fsPath}"`;
         const name = "JBang - " + uri.fsPath.substring(uri.fsPath.lastIndexOf("/") + 1);
         let terminalOptions = { name } as ITerminalOptions;
         return await terminalCommandRunner.runInTerminal(command, terminalOptions);
@@ -25,7 +25,7 @@ export class JBangRunner {
 
     public async exportNative(uri: Uri): Promise<Terminal> {
         const exec = jbang();
-        const command = `${exec} export native ${uri.fsPath}`;
+        const command = `${exec} export native "${uri.fsPath}"`;
         const name = "JBang - " + uri.fsPath.substring(uri.fsPath.lastIndexOf("/") + 1);
         let terminalOptions = { name } as ITerminalOptions;
         return await terminalCommandRunner.runInTerminal(command, terminalOptions);

